21/README.md

```diff
@@ -41,41 +41,33 @@ Consider a network of 4 nodes `{A,B,C,D}`. If a node `A` wishes to add a new nod
 
   - The `VALIDATOR_ELECTION` transaction is signed with the private key of the validator proposing the election. The private key is generated by Tendermint, and stored in the `priv_validator.json` in the Tendermint's `$TMHOME` directory (usually located at `~/.tendermint/`).
   - `E_POWER` should be such that `0 < E_POWER < TOTAL_POWER/3` where `TOTAL_POWER` denotes the total voting power of the current network.
+  - `E_NODE_ID` can be generated by executing `tendermint show_node_id` in the shell of the node
 
 
 2. The `election_id` is then manually sent (via email or message) to all members of the network.
 
-3. The node operator can list the contents of `election_id` request using,
+3. The node operator can list the contents of `election_id` request using [`bigchaindb election show ...`](../18),
 
   ```
-  $ bigchaindb upsert-validator show <election_id>
+  $ bigchaindb election show <election_id>
   public_key=Wn2DedV9OA0LJJjOxr7Sl7jqCSYjQihA6dCBX+iHaEI=
   power=10
   node_id=82190eb6396bdd80b83aef0f931d0f45738ed075
+  status=ONGOING
   ```
 
-  The above command list the details about the node which is being added/updated/deleted from the network.
-
 4. If the node operator aggrees to the operation being proposed by the `election_id` then they can vote on the same using the following,
 
   ```
-  $ bigchaindb upsert-validator approve <election_id> --private-key /home/user/.tendermint/config/priv_validator.json
+  $ bigchaindb election approve <election_id> --private-key /home/user/.tendermint/config/priv_validator.json
   ```
 
-  The above command `POST`s a [`VALIDATOR_ELECTION_VOTE`][spec_validator_election_vote] transaction casting the vote of the node for given `election_id`.
-  NOTE: The `VALIDATOR_ELECTION_VOTE` transaction is signed using the private key generated and stored by Tendermint in `priv_validator.json`.
-
-5. Node operators can check the status of the election as follows,
+  The above command `POST`s a [`ELECTION_VOTE`][spec_election_vote] transaction casting the vote of the node for given `election_id`.
+  NOTE: The `ELECTION_VOTE` transaction is signed using the private key generated and stored by Tendermint in `priv_validator.json`.
```

Comment:
The vote transaction operation is called `VOTE`.

Reply:
I will fix that as well.